### PR TITLE
add scripts to find and retrigger failed renewal enrollments

### PR DESCRIPTION
**Pull request by RyanEddyIC**
# PR Checklist

Please check if your PR fulfills the following requirements
- [x] The title follows our [guidelines](https://github.com/ideacrew/enroll/blob/trunk/CONTRIBUTING.md#commit
)
- [ ] Tests for the changes have been added (for bugfixes / features)

# PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature (requires Feature flag)
- [x] Data fix or migration (inert code, no impact until run)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Dependency updates (e.g., add a new gem or update version)

# What is the ticket # detailing the issue?

Ticket: 
[186116591](https://www.pivotaltracker.com/n/projects/2640057/stories/186116591)

# A brief description of the changes

Current behavior:

New behavior:
Scripts added to identify failed renewal enrollments and attempt to retrigger the renewal, outputting hbx_ids of failed enrollments and their primary members.

# Feature Flag

For all new feature development, a feature flag is required to control the exposure of the feature to our end users. A feature flag needs a corresponding environment variable that is used to initialize the state of the flag. Please share the name of the environment variable below that would enable/disable the feature and which client(s) it applies to.

Variable name:

- [ ] DC
- [ ] ME

# Additional Context
Include any additional context that may be relevant to the peer review process.

### CODE REVIEW

**Review by saikumar9 — DISMISSED** [review]
lgtm!

**Review by mdkaraman — DISMISSED** [review]

**Review by yosefserkez — DISMISSED** [review]
Looks good. Creates a mechanism for re-triggering failed renewals exclusive of existing ones.

File: script/renew_targeted_ivl_enrollments.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+# This script will attempt to call RenewEnrollment on previously failed enrollments
+# and output a CSV of all enrollments still failing.
+
+unless ARGV[0].present?
+  puts "Please include the year to retrigger renewals for (e.g. 2023)" unless Rails.env.test?
+  exit
+end
+require 'csv'
+year = ARGV[0]
```
Comment (4till2) [inline script/renew_targeted_ivl_enrollments.rb:10]: year = ARGV[0].to_i

**Review by mdkaraman — APPROVED** [review]
Looks good!

**Review by yosefserkez — APPROVED** [review]

**Pull request merged**